src/card.css

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__title {
+  margin: 0 0 4px;
```

Comment:
You can use global reset of the margin for the h2, 
+ after set for .card__title only margin-bottom style